### ISSUE TITLE
Deprecated dependency

### ISSUE THREAD

**Issue by ThomasSousa96** [payara/Payara#2862]
# Description #
----------

Running the payara/micro:5.182 show some warns 

## Expected Outcome

Use non deprecated dependencies

## Current Outcome

```
[2018-06-18T13:08:41.191+0000] [] [WARN] [] [org.jboss.weld.Bootstrap] [tid: _ThreadID=1 _ThreadName=main] [timeMillis: 1529327321191] [levelValue: 900] WELD-000146: BeforeBeanDiscovery.addAnnotatedType(AnnotatedType<?>) used for class org.glassfish.jersey.ext.cdi1x.transaction.internal.WebAppExceptionHolder is deprecated from CDI 1.1!

[2018-06-18T13:08:41.195+0000] [] [WARN] [] [org.jboss.weld.Bootstrap] [tid: _ThreadID=1 _ThreadName=main] [timeMillis: 1529327321195] [levelValue: 900] WELD-000146: BeforeBeanDiscovery.addAnnotatedType(AnnotatedType<?>) used for class org.glassfish.jersey.ext.cdi1x.transaction.internal.WebAppExceptionInterceptor is deprecated from CDI 1.1!

[2018-06-18T13:08:41.197+0000] [] [WARN] [] [org.jboss.weld.Bootstrap] [tid: _ThreadID=1 _ThreadName=main] [timeMillis: 1529327321197] [levelValue: 900] WELD-000146: BeforeBeanDiscovery.addAnnotatedType(AnnotatedType<?>) used for class org.glassfish.jersey.ext.cdi1x.transaction.internal.TransactionalExceptionMapper is deprecated from CDI 1.1!

[2018-06-18T13:08:41.316+0000] [] [WARN] [] [org.jboss.weld.Bootstrap] [tid: _ThreadID=1 _ThreadName=main] [timeMillis: 1529327321316] [levelValue: 900] WELD-000146: BeforeBeanDiscovery.addAnnotatedType(AnnotatedType<?>) used for class org.glassfish.jersey.ext.cdi1x.servlet.internal.CdiExternalRequestScope is deprecated from CDI 1.1!

[2018-06-18T13:08:41.337+0000] [] [WARN] [] [org.jboss.weld.Bootstrap] [tid: _ThreadID=1 _ThreadName=main] [timeMillis: 1529327321337] [levelValue: 900] WELD-000146: BeforeBeanDiscovery.addAnnotatedType(AnnotatedType<?>) used for class org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider$JaxRsParamProducer is deprecated from CDI 1.1!

```

## Steps to reproduce (Only for bug reports) 

Run the docker image using the below command

1 -**  Start the domain

```
docker run --rm -it -v ~/example.war:/opt/payara/deployments/example.war payara/micro:5.182
```

**Comment by smillidge** [comment]
Yes these are from Jersey which needs some work to remove old CDI semantics in some areas.

**Comment by smillidge** [comment]
Fixed in 5.183
